src/client/utils/dom.js

```diff
@@ -110,7 +110,18 @@ export function getActiveElement (currentDocument) {
     // https://github.com/DevExpress/testcafe-hammerhead/issues/768
     var doc = currentDocument || document;
 
-    return isDomElement(doc.activeElement) ? doc.activeElement : doc.body;
+    var el = isDomElement(doc.activeElement) ? doc.activeElement : doc.body;
+
+    while (el && el.shadowRoot) {
```

Comment:
Is it possible to create nested shadowRoots?
Can active element be not `isDomElement`?

If it is true, then add tests for following cases.